src/Navigation/Dropdown/Dropdown.tsx

```diff
@@ -71,8 +71,7 @@ const Dropdown = (props: Props) => {
     const itemElement = e.target as HTMLLIElement;
 
     if (itemElement.dataset.value) {
-      // TODO: This should probably set to itemElement.innerHTML
-      setDropdownLabel(itemElement.dataset.value);
+      setDropdownLabel(itemElement.innerHTML);
```

Comment:
shall we move `setDropdownLabel(itemElement.innerHTML)` out of this `if` block? I think the dropdown label should always be updated whether `itemElement.dataset.value` has value or not. What do you think?

Reply:
Oh actually I think this would break some usages of this component because people have been using it as a dropdown menu where clicking an option will trigger an action without changing the label.